admin/controllers/restart.js

```diff
@@ -60,7 +60,8 @@ controller.postSubmitRestartList = async (req, res, next) => {
   }
   res.locals.pageTitle = 'Restarts'
   req.breadcrumbs(res.locals.pageTitle)
-  req.flash('info', `${submittedPupils.length} new pupils have been submitted for restart`)
+  const restartInfo = submittedPupils.length < 2 ? '1 pupil' : `${submittedPupils.length} pupils`
```

Comment:
I think it makes sense to test for equality here: `submittedPupils.length === 1`

0 pupils
1 pupil
2 pupils
...